tokio/src/runtime/builder.rs

```diff
@@ -67,32 +65,37 @@ pub struct Builder {
     /// To run before each worker thread stops
     pub(super) before_stop: Option<Callback>,
 
-    #[cfg(feature = "blocking")]
-    #[cfg_attr(docsrs, doc(cfg(feature = "blocking")))]
     /// Customizable keep alive timeout for BlockingPool
     pub(super) keep_alive: Option<Duration>,
 }
 
 pub(crate) type ThreadNameFn = std::sync::Arc<dyn Fn() -> String + Send + Sync + 'static>;
 
-#[derive(Debug, Clone, Copy)]
-enum Kind {
-    Shell,
-    #[cfg(feature = "rt-core")]
-    Basic,
+pub(crate) enum Kind {
```

Comment:
So no single thread runtime? or are we punting that?

Reply:
No single thread for now IMO. We can do that post. It's a *lot* easier to do a single-thread by hand now as you can throw `Runtime` in an arc.